pkg/utils/utils.go

```diff
@@ -111,3 +111,19 @@ func IsPrefixSupported(input string, prefixes []string) bool {
 	}
 	return false
 }
+
+func AppendEnvVarIfNotExists(slice []v1.EnvVar, elems ...v1.EnvVar) []v1.EnvVar {
+	for _, item := range elems {
+		appendEnvVarIfNotExists(slice, item)
+	}
+	return slice
+}
+
+func appendEnvVarIfNotExists(slice []v1.EnvVar, elem v1.EnvVar) []v1.EnvVar {
```

Comment:
seems like it is cleaner to combine the two functions?

Reply:
updated!